core/controllers/skill_editor.py

```diff
@@ -282,6 +282,14 @@ def get(self, comma_separated_skill_ids):
 class FetchSkillsHandler(base.BaseHandler):
     """A handler for accessing all skills data."""
 
+    URL_PATH_ARGS_SCHEMAS = {}
+    HANDLER_ARGS_SCHEMAS = {
+        'PUT': {},
```

Comment:
Why is this added?

Reply:
As it was instructed in the docs of  **How to write schemas** , if there is a handler with no arguments the schema should look like this to avoid `NotImplemented` error , I followed that and added these.

Reply:
Hi, @vojtechjelinek I am not getting the error here.
Could you please help me with what changes should be done here?

Reply:
But PUT is not in this class, so you don't need to add it. The only handler that is defined is the `get` so that is the only thing to add.

Reply:
Ok, I got it! 
 I missed that out, Thanks a lot !!!